src/extension/params-extractor/src/test/java/org/geoserver/params/extractor/TestSupport.java

```diff
@@ -59,18 +58,12 @@ private void deleteTestDirectoryQuietly() {
         }
     }
 
-    protected static void doWork(String resourcePath, Consumer<InputStream> consumer)
```

Comment:
simplified this a bit. This avoids too excessive lambda expressions and significantly improves the readability.

OK, not strictly related to the original topic; split this off into a separate PR?

Reply:
I think it is fine here; it is part of your theme of trying to make resource access less difficult; and TestSupport is not an api change for the web application